### PR TITLE
fix(rgb++): loading picture when loading

### PR DESCRIPTION
**Pull request by Daryl-L**
https://github.com/Magickbase/ckb-explorer-public-issues/issues/712

/address/tb1qssu8pscsuevnmvhnr08840wvuzdan27k9m46ja

image

image

### CODE REVIEW

**Comment by Keith-CY** [comment]
How do you think about the loading animation? @Sven-TBD

**Comment by Kirl70** [comment]
We can use any of the first three links from this source, just change the color to match the explorer theme color.


https://codepen.io/aurer/pen/ZEJxpO
image

**Comment by Keith-CY** [comment]
> We can use any of the first three links from this source, just change the color to match the explorer theme color.
> 
> [codepen.io/aurer/pen/ZEJxpO](https://codepen.io/aurer/pen/ZEJxpO) image

@Sven-TBD please pick one for the loading of the dialog

**Comment by Keith-CY** [comment]
Let's use the 3rd one

image

**Comment by Keith-CY** [comment]
> Let's use the 3rd one
> 
> image

Any update on the animation? @Daryl-L

**Comment by FrederLu** [comment]
https://ckb-explorer-frontend-in-magickbase-repo-7mrdo8pp3-magickbase.vercel.app/xudt/0x5d456d75a00f65d5aed31c180939294464bb158e4c246cf02e62ac6c7fbb4b43
image

The loading state and its styles are implemented.

---

image

But the second point here has not been processed yet, and there is a `RGB++ Digest` pop-up window.

image
image

---
Expected style：
image
image

**Comment by FrederLu** [comment]
image
image

The position of the `RGB++ Digest` pop-up window on PC has been adjusted, but the problem still exists on mobile devices. Please confirm.  @Daryl-L

File: src/components/AwesomeLoadings/SmallSpinner/index.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import styles from './styles.module.scss'
+import { ReactComponent as SmallSpinner } from '../../../assets/small-spinner.svg'
+
+const Loading = () => {
+  return (
+    <div className={styles.container}>
+      <SmallSpinner />
+    </div>
```
Comment (Keith-CY) [inline src/components/AwesomeLoadings/SmallSpinner/index.tsx:8]: Why not add the class name on `SmallSpinner` directly?

**Review by Keith-CY — APPROVED** [review]

**Review by FrederLu — APPROVED** [review]

**Pull request merged**